### PR TITLE
Checks execution policy addendum

### PR DESCRIPTION
**Pull request by CDimonaco**
# Description

This PR adds the policy guard to the check execution button in host/cluster details.

immagine

## How was this tested?

Automatic tests

### CODE REVIEW

File: assets/js/pages/HostDetailsPage/HostDetails.test.jsx [hunk 1]
```diff
@@ -346,18 +373,62 @@ describe('HostDetails component', () => {
...
+      );
 
       expect(
         screen.getByText('No check results available.')
       ).toBeInTheDocument();
     });
   });
+
+  describe('forbidden actions', () => {
+    it('should not show start execution button when the user abilities are not compatible', async () => {
```
Comment (arbulu89) [inline assets/js/pages/HostDetailsPage/HostDetails.test.jsx:398]: `should disable start execution button` better

File: assets/js/pages/HostDetailsPage/HostDetails.test.jsx [hunk 2]
```diff
@@ -346,18 +373,62 @@ describe('HostDetails component', () => {
...
+      const startExecutionButton = screen.getByText('Start Execution');
+      expect(startExecutionButton).toBeDisabled();
+
+      await user.hover(startExecutionButton);
+      expect(
+        screen.queryByText('You are not authorized for this action')
+      ).toBeInTheDocument();
+    });
+
+    it('should show start execution button when the user abilities are compatible', () => {
```
Comment (arbulu89) [inline assets/js/pages/HostDetailsPage/HostDetails.test.jsx:419]: `should enable start execution...`

**Review by arbulu89 — APPROVED** [review]
Nice

File: assets/js/pages/HostDetailsPage/HostDetails.test.jsx (hunk 2 above)
Comment (CDimonaco) [reply assets/js/pages/HostDetailsPage/HostDetails.test.jsx]: done!

**Pull request merged**